src/main/java/net/sf/jabref/logic/autocompleter/AbstractAutoCompleter.java

```diff
@@ -58,9 +58,12 @@ public AbstractAutoCompleter(AutoCompletePreferences preferences) {
      * Otherwise the completion is case insensitive.
      */
     @Override
-    public String[] complete(String toComplete) {
+    public List<String> complete(String toComplete) {
+        if(toComplete == null) {
+            return new ArrayList<>();
```

Comment:
Yes, I wondered about that too. In the end I have chosen the new list since I saw no reason to prohibit that the caller adds further strings afterwards.